ivre/xmlnmap.py

```diff
@@ -470,22 +491,27 @@ def endElement(self, name):
                                      "empty, got [%r]\n" % self._curtablepath)
                 self._curscript[infokey] = self._curtable
                 self._curtable = {}
-            elif infokey != 'infos' and infokey in ADD_TABLE_ELEMS:
-                infos = ADD_TABLE_ELEMS[infokey]
-                if type(infos) is utils.REGEXP_T:
-                    infos = infos.search(self._curscript.get('output', ''))
-                    if infos is not None:
-                        infosdict = infos.groupdict()
-                        if infosdict:
-                            self._curscript[infokey] = infosdict
-                        else:
-                            infos = list(infos.groups())
-                            if infos:
-                                self._curscript[infokey] = infos
-                elif hasattr(infos, "__call__"):
-                    infos = infos(self._curscript)
-                    if infos is not None:
-                        self._curscript[infokey] = infos
+            elif infokey != 'infos':
+                if infokey in ADD_TABLE_ELEMS:
+                    infos = ADD_TABLE_ELEMS[infokey]
+                    if type(infos) is utils.REGEXP_T:
```

Comment:
Generally, it is better to use `isinstance` instead of `type(..) is ..`.
(Point `Object type comparisons should always use isinstance() instead of comparing types directly. ` in [PEP08](https://www.python.org/dev/peps/pep-0008/))